docs/additional-postgres-setup.md

```diff
@@ -1,16 +1,18 @@
 # Setup your application with PostgreSQL
+
 Follow these links for detailed the installation guide.
```

Comment:
This line doesn't sound right.

Reply:
You're right it doesn't, although I wasn't looking at what was written, just running prettier. I can update that line in the text though.

Reply:
Oh I was oblivious to the fact that this was not one of your changes, my bad :(

Reply:
No worries ;)